openpype/hosts/houdini/plugins/publish/collect_fbx_type.py

```diff
@@ -0,0 +1,59 @@
+"""Collector for filmboxfbx types.
+
+Collectors act as a pre process for the validation stage.
+It is used mainly to update instance.data
+
+P.S.
+    There are some collectors that run by default
+    for all types.
+
+This plugin is part of publish process guide.
+"""
+import pyblish.api
+
+
+class CollectFilmboxfbxType(pyblish.api.InstancePlugin):
+    """Collect data type for fbx instance."""
+
+    hosts = ["houdini"]
+    families = ["fbx", "staticMesh"]
+    label = "Collect type of fbx"
+
+    # Usually you will use this value as default
+    order = pyblish.api.CollectorOrder
+
+    # overrides InstancePlugin.process()
+    def process(self, instance):
+
+        if instance.data["creator_identifier"] == "io.openpype.creators.houdini.unrealstaticmesh":  # noqa: E501
+            # such a condition can be used to differentiate between
+            #  instances by identifier becuase sometimes instances
+            #  may have the same family but different identifier
+            #  e.g. bgeo and alembic
+            instance.data["families"] += ["fbx"]
```

Comment:
Revisiting to the same staticmesh family in maya, staticmesh family extracts both abc and fbx format. If this family in Houdini acts similarly to Maya, it should be adding two families: `fbx` and `abc`. (You can find the codes of staticmesh family in Maya Hosts)
Maybe consider to rename this collector to `collect_unreal_staticmesh` and remove fbx as families in line 19.  (Otherwise, you are adding two `fbx` into families.)
```suggestion
        if instance.data["creator_identifier"] == "io.openpype.creators.houdini.unrealstaticmesh":  # noqa: E501
            # such a condition can be used to differentiate between
            #  instances by identifier becuase sometimes instances
            #  may have the same family but different identifier
            #  e.g. fbx and alembic
            instance.data["families"] += ["fbx"]
            instance.data["families"] += ["abc"]
```

Reply:
I think it might make it harder to handle collision prefixes (which basically checking path attribute values). 
anyways, I will revisit the  same staticmesh family in maya to learn more. 

I'll appreciate any recommendations.

Reply:
> I think it might make it harder to handle collision prefixes (which basically checking path attribute values). anyways, I will revisit the same staticmesh family in maya to learn more.
> 
> I wonder if it's possible to publish same data in to representations in houdini ? In maya it can publish the same set with different commands, however in houdini we should create a rop for each representation? so, is it possible for a publish instance to have two rops ?
> 
> I'll appreciate any recommendations. @moonyuet @BigRoy

Take reference from pointcache family in Houdini. It is two creators but same families. But It's only my suggestion. You can always do the abc extractor later if there is any user wanted in staticmesh family.

Reply:
> so, is it possible for a publish instance to have two rops ?

I don't know what will be the use case for that in production 

I can think of two options : 
1. by relying on subnets with several instances inside, each instance can have a different variant post fix ? 
2. Create multiple instances with different variants ? 

Fool Test: *Family was randomly picked*
![image](https://github.com/ynput/OpenPype/assets/20871534/d90ab974-02ad-4eb7-9e3a-70f59cf8b8d7)

Reply:
I will save this note!